pkg/addon/manager/addon.go

```diff
@@ -52,7 +52,7 @@ func (m *managedServiceAccountAddonAgent) GetAgentAddonOptions() agent.AgentAddo
 			CSRApproveCheck:   agent.ApprovalAllCSRs,
 			PermissionConfig:  m.setupPermission,
 		},
-		InstallStrategy: agent.InstallAllStrategy(common.AddonAgentInstallNamespace),
+		InstallStrategy: nil,
```

Comment:
We should introduce a flag for this

Reply:
i think so, let's add an flag

Reply:
Good plan, I will add a flag 

What should be the default behavior?

Reply:
Let's set the default behavior as-is

Reply:
updated the default value in the chart